### PR TITLE
Docs Update (python version & T2I

### PR DESCRIPTION
**Pull request by Millu**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [ ] Yes
- [X] No, because:

      
## Have you updated all relevant documentation?
- [X] Yes
- [ ] No

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

File: .github/workflows/test-invoke-pip.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ jobs:
         python-version:
           # - '3.9'
           - '3.10'
+          - '3.11'
```
Comment (psychedelicious) [inline .github/workflows/test-invoke-pip.yml:26]: hmm, this means all the tests need to run twice - do we have any concerns about using up our runner minutes?

File: docs/features/CONTROLNET.md [hunk 2]
```diff
@@ -17,52 +17,46 @@ image generation, providing you with a way to direct the network
...
-Command-line users can launch the model installer using the command
-`invokeai-model-install`.
 
 _Be aware that some ControlNet models require additional code
 functionality in order to work properly, so just installing a
 third-party ControlNet model may not have the desired effect._ Please
 read and follow the documentation for installing a third party model
 not currently included among InvokeAI's default list.
 
+Currently InvokeAI **only** supports "diffuser" style ControlNet models. These are
```
Comment (psychedelicious) [inline docs/features/CONTROLNET.md:49]:
```suggestion
Currently InvokeAI **only** supports 🤗 Diffusers-format ControlNet models. These are
```

File: docs/features/CONTROLNET.md [hunk 3]
```diff
@@ -17,52 +17,46 @@ image generation, providing you with a way to direct the network
...
+folders that contain the files `config.json` and/or
+`diffusion_pytorch_model.safetensors` and
+`diffusion_pytorch_model.fp16.safetensors`. The name of the folder is
+the name of the model.
+
+Diffuser-style ControlNet models are available at HuggingFace
```
Comment (psychedelicious) [inline docs/features/CONTROLNET.md:55]:
```suggestion
🤗 Diffusers-format ControlNet models are available at HuggingFace
```

File: docs/features/CONTROLNET.md [hunk 4]
```diff
@@ -133,14 +127,38 @@ Start/End  - 0 represents the start of the generation, 1 represents the end. The
...
+#### Usage
+Each T2I Adapter has two settings that are applied.
+
+Weight - Strength of the model applied to the generation for the section, defined by start/end.
+
+Start/End  - 0 represents the start of the generation, 1 represents the end. The Start/end setting controls what steps during the generation process have the ControlNet applied.
+
+Additionally, each  section can be expanded with the "Show Advanced" button in order to manipulate settings for the image pre-processor that adjusts your uploaded image before using it in during the generation process.
+
+**Note:** T2I-Adapter models and ControlNet models cannot be used together.
```
Comment (psychedelicious) [inline docs/features/CONTROLNET.md:153]:
```suggestion
**Note:** T2I-Adapter models and ControlNet models cannot currently be used together.
```

**Review by psychedelicious — CHANGES_REQUESTED** [review]

File: .github/workflows/test-invoke-pip.yml (hunk 1 above)
Comment (Millu) [reply .github/workflows/test-invoke-pip.yml:26]: Hmm good question - we can have tests for only 3.10 for the time being

**Review by psychedelicious — APPROVED** [review]

**Review by hipsterusername — APPROVED** [review]

**Pull request merged**